### ISSUE TITLE
notebook.showCellStatusBar no longer respected

### ISSUE THREAD

**Issue by JuanCab** [microsoft/vscode#236181]
Type: <b>Bug</b>

Attempts to run a cell not longer respect the showCellStatusBar being set to visible, seem to default to visibleExecute, since the cell's status bar disappears when execution completes.

Extension version: 2024.11.0
VS Code version: Code 1.96.0 (Universal) (138f619c86f1199955d53b4166bef66ef252935c, 2024-12-11T02:29:09.626Z)
OS version: Darwin arm64 24.2.0
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M3 (8 x 2400)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|5, 4, 4|
|Memory (System)|24.00GB (0.32GB free)|
|Process Argv|--crash-reporter-id d9f43910-1656-4091-bdb2-243e9264914d|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscod805cf:30301675
binariesv615:30325510
vsaa593cf:30376535
py29gd2263:31024239
vscaat:30438848
c4g48928:30535728
azure-dev_surveyone:30548225
vscrp:30673768
962ge761:30959799
pythonnoceb:30805159
pythonmypyd1:30879173
2e7ec940:31000449
pythontbext0:30879054
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
dsvsc021:30996838
dvdeprecation:31068756
dwnewjupyter:31046869
2f103344:31071589
nativerepl2:31139839
pythonrstrctxt:31112756
nativeloc1:31192215
cf971741:31144450
iacca1:31171482
notype1:31157159
5fd0e150:31155592
dwcopilot:31170013
stablechunks:31184530
6074i472:31201624

```

</details>

**Comment by JuanCab** [comment]
Here's a gif showing this behavior in action, notice the cell status appears and shows the execution while the cell is running and then immediately disappears.
![Image](https://github.com/user-attachments/assets/8ee2a34f-d006-4d5b-b51e-edde7dad2bb5)

Here's the funny thing, moments later I tried to re-capture the screen recording to show the showCellStatusBar setting to show it was set to "visible" and while I confirmed it was, the cell status did NOT disappear on that recording!  So this bug appears to be an intermittent problem or something "complex", being caused by a second condition which I am not aware of.  In any case, there it is, a recording of disappearing cellStatusBar information.
